src/embed/connection.rs

```diff
@@ -60,6 +57,7 @@ async fn connection_future<'a, 'b>(
         Some(v) => v,
     };
     match req.rpc.as_str() {
+        "getRoot" => execute(do_get_root, store, txns, req).await,
```

Comment:
[cbb] no need to go back and address but so many places i see lists growing that are not in alpha order. makes it easier to find the thing if they are in order.

Reply:
good point, will alpha next time i'm there.